pages/docs/Style Guide.md

```diff
@@ -0,0 +1,108 @@
+
+- Use commas before the seven, so-called, coordinating conjunctions (and, but, so, for, nor, yet, or) to separate two independent clauses.
+  - ❌ **Not this:** There are 4 ways you can add your profile **but** for this Quickstart, we will use the GitHub UI.
+  - ✅ **Use this:** There are 4 ways you can add your profile, **but** for this Quickstart, we will use the GitHub UI.
+
+### Capitalisation
+
+- Only capitalize terms that describe a product, its feature, the first letter of a sentence, or a person’s name.
+  - ❌ **Not this:** There are 4 ways you can add your profile but****for this **quickstart**we will use the **github ui**.
+  - ✅ **Use this:** There are 4 ways you can add your profile, **but** for this Quickstart, we will use the **GitHub UI**.
+
+## Clarity 🤔
+
+Clarity in writing helps ensure that your audience understands your message and intentions. Here are some best practices to follow:
+
+### Active vs. Passive
+
+Using the active voice in writing makes your message more direct and engaging.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+- ✅ **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+### Jargon
+
+Using technical jargon can alienate non-technical readers.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+- ✅ **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+## Accessibility ♿
+
+### Bullet Points
+
+- Avoid describing things only by their color or position.
+  - ❌ **Not this:** see **the** **image** above.
+  - ✅ **Use this:** See **Image 2.2**.
+
+### Emojis
+
+- Avoid using emojis as bullet points or numbered lists.
+- ❌ **Not this:** 1️⃣ Fork the repository
+- 2️⃣ Visit the Accessibleforall repository
+  - ✅ **Use this:** **1.** Fork the repository
+- **2️.** Visit the Accessibleforall repository
+
+- Avoid using emojis in the middle of a sentence.
+  - ❌  **Not this:** There are 4️⃣ ways you can add your profile, but for this Quickstart we will use the GitHub UI.
+- ✅ **Use this:** There are **4** ways you can fix your profile, but for this tutorial, we will use the GitHub UI.
+
+- Use them sparingly at the end of a sentence.
+  - ❌ **Not this:** Join the conversation on our Discord community! 😀
+  - ✅ **Use this:** Join the conversation on our Discord community!
+
+### Headings
+
+- Use descriptive titles for headings.
+  - ❌ **Not this:** `<h1> Yoga</h1>`
+- ✅ **Use this:** `<h1>Yoga for Developers</h1>`
+
+- Place headings in chronological order.
+- ❌ **Not this:** `<h1>Yoga for Developers</h1>`
+  `<h2>What is Yoga?</h2>`
```

Comment:
I think two `h2` can follow each other but it should not be: 

```HTML
<h1>Heading one</h1>
<h3>Heading one</h3>
<h2>Heading 2</h2>
```

Reply:
Noted